packages/fast-form-generator-react/src/form/form.style.ts

```diff
@@ -0,0 +1,31 @@
+import { toPx } from "@microsoft/fast-jss-utilities";
+import {  applyCleanListStyle } from "../utilities/form-input.style";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { IFormClassNameContract } from "../class-name-contracts/";
+
+const styles: ComponentStyles<IFormClassNameContract, {}> = {
+    form_breadcrumbs: {
+        display: "flex",
+        flexWrap: "wrap",
+        marginTop: toPx(4),
+        paddingBottom: toPx(24),
+        ...applyCleanListStyle(),
+        "& li": {
+            display: "inline-block",
+            paddingRight: toPx(8),
+            "&:after": {
+                content: "'/'",
+                paddingLeft: toPx(8)
+            },
+            "&:last-child:after": {
+                content: "''",
+                paddingLeft: "0"
+            },
+            "& a": {
+                color: "#0078D7"
```

Comment:
can this be one of the colors from your styles export? Looks like its just about the same as colors.blue in fact, I think we could use it for now

Reply:
Fixed.